L/LAMMPS/build_tarballs.jl

```diff
@@ -49,6 +50,8 @@ platforms = expand_gfortran_versions(platforms)
 # libgfortran3 does not support `!GCC$ ATTRIBUTES NO_ARG_CHECK`. (We
 # could in principle build without Fortran support there.)
 platforms = filter(p -> libgfortran_version(p) ≠ v"3", platforms)
+# Compiler failure
+platforms = filter(p -> p != Platform("aarch64", "linux"; libc="gnu", libgfortran_version=v"4") , platforms)
```

Comment:
He, platform matching is hard
```suggestion
filter!(p -> !(Sys.islinux(p) && arch(p) == "aarch64" && libc(p) =="glibc" && libgfortran_version(p) == v"4") , platforms)
```